app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const app = express()
+/* don't need app.use(express.json()) yet as express does this, only need with post or patch requests*/
```

Comment:
Nice to see this here Sula, but I'd probably recommend you remove comments like this before submitting official pull requests in future